assets/src/edit-story/components/panels/panel/panel.js

```diff
@@ -44,12 +45,41 @@ function Panel({
   ariaLabel = null,
   ariaHidden = false,
 }) {
-  const [isCollapsed, setIsCollapsed] = useState(false);
-  const [expandToHeight, setExpandToHeight] = useState(initialHeight);
-  const [height, setHeight] = useState(initialHeight);
+  const persisted = useMemo(() => {
+    let parsed = null;
+    try {
+      const stored = localStorage.getItem(
+        `${LOCAL_STORAGE_PREFIX.PANEL}:${name}`,
+        'null'
+      );
+      parsed = JSON.parse(stored);
+    } catch (e) {
+      // @TODO Add some error handling.
```

Comment:
Hmm `setInitialBannerPreviouslyClosed` in `useTelemetryOptIn` should probably also wrap this in a try/catch

cc @dmmulroy

Reply:
Sounds good, I'll update that right now - @littlemilkstudio what is the second argument in `getItem` for?  As far as I can tell `getItem` only accepts one argument?

Reply:
I have no problem updating it, but what error is this catching? It seems like exceptions can be potentially thrown if the user has prevented a site from using storage, but none of our other localstorage uses are wrapped in try/catch (see just below).  By default, localstorage will return null if a key does not exist which is fine to pass to JSON.parse.

Also, what is the second argument to `getItem`?  AFAIK it only accepts one.